contrib/gitian-builder/README.md

```diff
@@ -0,0 +1,114 @@
+<h1 align="center">
+Dogecoin Core [DOGE, Ð]  
+<br/><br/>
+<img src="https://static.tumblr.com/ppdj5y9/Ae9mxmxtp/300coin.png" alt="Dogecoin" width="300"/>
+</h1>
+
```

Comment:
What about instructions in https://github.com/dogecoin/dogecoin/blob/master/doc/gitian-building.md ?

Contrib folders have `README` but there is also some gitian instruction in `doc`. Those instructions in `doc` are outdated and do not work anymore. Where gitian instructions should go to avoid duplicate information, following DRY ?

Keep the `contrib/gitian-builder` README and add a link to it from the doc, maybe near a release process explanation ? And removing https://github.com/dogecoin/dogecoin/tree/master/doc/gitian-building folder with pictures + https://github.com/dogecoin/dogecoin/blob/master/doc/gitian-building.md ?

Reply:
I'm okay with the readme, just the branding feels redundant.

Reply:
Where the gitian instructions should go ? Docs or contrib ?

Reply:
Both. This readme should describe this script. The document in doc/ should describe the generic process regardless of this script but can mention and link it.